docs/user/subprojects.rst

```diff
@@ -1,43 +1,67 @@
 
-The documentation for ``Bar`` will be available under this same path:
+* Organizations that need all their projects visible in one documentation portal or landing page
+* Projects that document and release several packages or extensions
+* Organizations or projects that want to have a common search function for several sets of documentation
 
-https://foo.readthedocs.io/projects/bar/en/latest/
+For a main project ``example-project``, a subproject ``example-project-plugin`` can be made available as follows:
 
-Adding a subproject
--------------------
+* Main project: https://example-project.readthedocs.io/en/latest/
+* Subproject: https://example-project.readthedocs.io/projects/plugin/en/latest/
 
-In the admin dashboard for your project, select "Subprojects" from the menu.
-From this page you can add a subproject by typing in the project slug.
+.. seealso::
 
-Subproject aliases
-~~~~~~~~~~~~~~~~~~
+   :doc:`/guides/subprojects`
+     Learn how to create and manage subprojects
+
+   :doc:`/guides/intersphinx`
+     Learn how to use references between different Sphinx projects, for instance between subprojects
 
-You can use an alias for the subproject when it is created. This allows you to override the URL that is used to access it, giving more configurability to how you want to structure your projects.
 
 Sharing a custom domain
 -----------------------
 
-Projects and subprojects can also be used to share a custom domain with a number
-of projects. To configure this, one project should be established as the parent
-project. This project will be configured with a custom domain. Projects can then
-be added as subprojects to this parent project.
+Projects and subprojects can be used to share a custom domain.
+To configure this, one project should be established as the main project.
+This project will be configured with a custom domain.
+Other projects are then added as subprojects to the main project.
+
+If the example project ``example-project`` was set up with a custom domain,
+``docs.example.com``, the URLs for projects ``example-project`` and ``example-project-plugin`` with alias ``plugin`` would
+respectively be at:
+
+* ``example-project``: https://docs.example.com/en/latest/
+* ``example-project-plugin``: https://docs.example.com/projects/plugin/en/latest/
+
+Using aliases
+-------------
+
+You can choose an alias for the subproject when it is created.
+This allows you to override the URL that is used to access it,
+giving more control over how you want to structure your projects.
+
+You can set your subproject's project name and :term:`slug` however you want,
+but we suggest prefixing it with the name of the main project.
 
-If the example project ``Foo`` was set up with a custom domain,
-``docs.example.com``, the URLs for projects ``Foo`` and ``Bar`` would
-respectively be at: https://docs.example.com/en/latest/ and
-https://docs.example.com/projects/bar/en/latest/
+Typically, a subproject is created with a ``$mainproject-`` prefix,
```

Comment:
```suggestion
Typically, a subproject is created with a ``mainproject-`` prefix,
```

Is the `$` on purpose here?

Reply:
Yes, well, I am a bit clueless about how to write placeholders.

@ericholscher do you have a favorite? `<placeholder>` or `$placeholder` or `PLACEHOLDER` or some other variant?

Reply:
(would like to immediately add it to the style guide as well - this keeps popping up)

Reply:
I think `<Thing>` for something abstract (eg. `<Username dropdown>`) and `$mainproject` for a string substitution would be my initial thinking.

Reply:
I think we have been using `<thing>` in most of the pages in the documentation. The only place were we use `$` are redirects and outgoing webhooks --which is a more technical audience that may be more used to what `$` could mean.

That doesn't mean that `<thing>` is a lot clearer for non-technical audiences, but at least keeps consistency with other pages: https://docs.readthedocs.io/en/stable/hosting.html#subdomain-support and https://docs.readthedocs.io/en/latest/custom-domains.html

That said, I think we need a `<thing>` entry in our glossary explaining that `<>` means a substitution done by the user/reader 👍🏼

Reply:
I tend to agree with Manuel that maybe using `<thing>` more or less for everything is simpler but would like to see it as a list of examples.

For this article, it will make it consistent with the rest of the article... so we can _somewhat easily_ find these occurrences and update them after we've finished a decision in the Documentation Style Guide.

```suggestion
Typically, a subproject is created with a ``<mainproject>-`` prefix,
```

Reply:
Just flagging that I started typing in a style guide update, BRB...